project/Common.scala

```diff
@@ -35,10 +35,8 @@ object Common extends AutoPlugin {
       "-Wconf:cat=deprecation&msg=since Akka 2\\.6\\.:s",
       // tolerate deprecations from Akka HTTP 10.2.0 until Pekko 1.1.x where we clean up
       "-Wconf:cat=deprecation&msg=since Akka HTTP 10\\.2\\.:s",
-      "-Wconf:msg=reached max recursion depth:s") ++
-    (if (isJdk8) Seq.empty
-     else if (scalaBinaryVersion.value == "2.12") Seq("-target:jvm-1.8")
-     else Seq("-release", "8")),
```

Comment:
Here `-release 8` was added before. Are you saying that it makes a difference whether `-release:8` or `-release 8` is used?

Reply:
Yes the flag was actually updated and improved, the PR provides all of the details.